### PR TITLE
Add redux aa1 to implement 1

### PR DESCRIPTION
**Pull request by Aichnerc**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Documentation (non-breaking change which adds documentation)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently Redux is not implemented.

**What is the new behavior (if this is a feature change)?**
- Actions and reducers have been added and were successfully implemented.

### CODE REVIEW

File: src/store/actions/pipelineActions.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        domain: "gitlab.local",
+        org: "anexia",
+        title: "Skynet",
+        latestActivity: 1596034377,
+        isActive: true,
+      },
+      {
+        domain: "share.local",
+        org: "anexia",
+        title: "Hydra",
```
Comment (schettn) [inline src/store/actions/pipelineActions.js]: @Aichnerc Neh..

**Review by schettn — APPROVED** [review]
Looks good for now.
Amore intensive review will take place in the master merge.

Please remove all those stuff which SNEK doesn't stand for.. You know what I mean.

**Review by pinterid — APPROVED** [review]
Looks good.

File: src/store/actions/connectorActions.js [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+            statistics: {
+              memberOrigin: false,
+            },
+          },
+          contribData: {
+            commits: [{ total: 1 }],
+          },
+        },
+      },
+    ];
```
Comment (schettn) [inline src/store/actions/connectorActions.js:37]: The settings have to take care of whether data such as projects, user, etc are displayed or not. The dummy data must also correspond to this.

File: src/store/actions/connectorActions.js [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+          },
+        },
+      });
+    }
+  };
+};
+
+// Get connector by id
+export const getConnectorById = (id) => {
+  return (dispatch, getState, { clientShopify }) => {
```
Comment (schettn) [inline src/store/actions/connectorActions.js]: looks like shopify xD

File: src/store/actions/connectorActions.js (hunk 2 above)
Comment (schettn) [reply src/store/actions/connectorActions.js:37]: Adjust this in the whole file.

**Review by schettn — CHANGES_REQUESTED** [review]
There is some invalid stuff.. have a look.
Btw `clientShopify` is all over the code. I would change this xD

File: src/store/actions/connectorActions.js (hunk 2 above)
Comment (Aichnerc) [reply src/store/actions/connectorActions.js:37]: I'll adjust that on the go when implement actual dummy data functionality. I am aware of this, yes.

**Review by schettn — APPROVED** [review]
LGTM

**Pull request merged**